canvas_modules/harness/src/client/constants/properties-documentation-constants.js

```diff
@@ -1099,6 +1099,122 @@ _defineConstant("TWISTY_PANEL_PROPS_INFO", {
 		}
 	}
 });
+_defineConstant("TEARSHEET_PANEL_PROPS_INFO", {
+	"title": "Panel Type: Tearsheet",
+	"parameterDef": {
+		"titleDefinition": {
+			"title": "Group: Tearsheet",
+			"editable": false
+		},
+		"current_parameters": {
+			"code": "Age >= 55"
+		},
+		"parameters": [
+			{
+				"id": "code",
+				"type": "string"
+			}
+		],
+		"uihints": {
+			"id": "code.test",
+			"label": {
+				"default": "Code Test"
+			},
+			"editor_size": "medium",
+			"description": {
+				"default": "Test code controls"
+			},
+			"parameter_info": [{
+				"parameter_ref": "code",
+				"language": "text/x-python",
+				"enable_maximize": true,
+				"data": {
+					"tearsheet_ref": "tearsheet1"
+				},
+				"label": {
+					"default": "Code"
+				},
+				"description": {
+					"default": "Enter Python code"
+				},
+				"text_before": {
+					"default": "Press ctrl-space to autocomplete"
+				},
+				"control": "code",
+				"class_name": "code-control-class"
+			}],
+			"group_info": [{
+				"id": "tearsheet-code",
+				"label": {
+					"default": "Expression with Maximize"
+				},
+				"parameter_refs": [
+					"code"
+				]
+			}, {
+				"id": "tearsheet1",
+				"label": {
+					"default": "Python"
+				},
+				"description": {
+					"default": "Your change is automatically saved."
+				},
+				"type": "tearsheetPanel",
+				"group_info": [{
+					"id": "tearsheet-code-2",
+					"parameter_refs": [
+						"code"
+					]
+				}]
+			}]
+		},
+		"conditions": [],
+		"resources": {}
+	}
+});
+_defineConstant("TEARSHEET_PANEL_FROM_HOST_PROPS_INFO", {
+	"title": "Panel Type: Tearsheet",
+	"parameterDef": {
+		"titleDefinition": {
+			"title": "Group: Tearsheet",
+			"editable": false
+		},
+		"current_parameters": {},
+		"parameters": [],
```

Comment:
`titleDefinition`, `current_parameters`, `parameters`, `conditions`, `resources` - These sections can be removed. They're not displayed anywhere on UI after opening tearsheet.